drape_frontend/gui/gui_text.cpp

```diff
@@ -428,11 +425,8 @@ void MutableLabel::SetText(LabelResult & result, std::string text) const
   for (DynamicVertex & v : result.m_buffer)
   {
     v.m_normal += anchorModifyer;
-    result.m_boundRect.Add(m2::PointD(glsl::ToPoint(v.m_normal)));
+    result.m_boundRect.Add(glsl::ToPoint(v.m_normal));
   }
-
-  for (size_t i = result.m_buffer.size(); i < 4 * m_maxLength; ++i)
-    result.m_buffer.push_back(DynamicVertex(glsl::vec2(0.0, 0.0), glsl::vec2(0.0, 0.0)));
```

Comment:
Hm, sure here?

Reply:
This is a code for a static label (ruler, osm credits, debug fps string). I didn't see any difference in rendering and didn't find any reason why empty vertexes were added. It looks like an ancient artifact.